mmdet/datasets/transforms/loading.py

```diff
@@ -525,15 +525,16 @@ def __init__(
                 'panopticapi.git.')
         self.rgb2id = utils.rgb2id
 
+        self.file_client_args = file_client_args
+        self.file_client = FileClient(**file_client_args)
```

Comment:
The fix seems to be wierd. LoadAnnotations accept file_client_args, why do we initialize it here separately?

Reply:
Because in mmcv2.0.0rc4: 
In LoadAnnotations, it's not  initialize file_client
        if file_client_args is not None:
            warnings.warn(
                '"file_client_args" will be deprecated in future. '
                'Please use "backend_args" instead', DeprecationWarning)
            if backend_args is not None:
                raise ValueError(
                    '"file_client_args" and "backend_args" cannot be set '
                    'at the same time.')

            self.file_client_args = file_client_args.copy()
        if backend_args is not None:
            self.backend_args = backend_args.copy() 
But mmcv2.0.0rc3:
In LoadAnnotations, file_client is initialized
        self.file_client_args = file_client_args.copy()
        self.file_client = mmengine.FileClient(**self.file_client_args)

Reply:
Because in mmcv2.0.0rc4: 
In LoadAnnotations, it's not  initialize file_client
       if file_client_args is not None:
            warnings.warn(
                '"file_client_args" will be deprecated in future. '
                'Please use "backend_args" instead', DeprecationWarning)
            if backend_args is not None:
                raise ValueError(
                    '"file_client_args" and "backend_args" cannot be set '
                    'at the same time.')
            self.file_client_args = file_client_args.copy()
        if backend_args is not None:
            self.backend_args = backend_args.copy() 
But mmcv2.0.0rc3:
In LoadAnnotations, file_client is initialized
        self.file_client_args = file_client_args.copy()
        self.file_client = mmengine.FileClient(**self.file_client_args)